back-end/student-tracker-template.yaml

```diff
@@ -39,6 +39,33 @@ Resources:
       Role: !GetAtt IAMRole.Arn
     DependsOn:
       - DynamoDBTable
+  CoursesFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        ZipFile: |
+          const AWS = require('aws-sdk');
+          const docClient = new AWS.DynamoDB.DocumentClient();
+          const params = {
+           TableName: !Sub "next-byte-Courses-${Env}",
+           KeyConditionExpression: 'my-partition-key = :pk',
+           ExpressionAttributeValues: {
+            ':pk': 'my-partition-key-value'
+           }
+          };
+          docClient.query(params, function(err, data) {
+           if (err) {
+             console.log(err);
+           } else {
+             console.log(data.Items);
```

Comment:
I think the data should be consumed in a try-catch block, rather than console.log. console,log may not be suitable in a production environment like this